TestResults/Drools/7.6.0.Final/tck_results.properties

```diff
@@ -5,3 +5,4 @@ product.url=http://www.drools.org
 product.version=7.6.0.Final
 product.comment=Drools provides a compliance level 3 runtime DMN engine
 last.update=2018-02-22
+tck.runner.manual.url=https://github.com/dmn-tck/tck/tree/master/runners/dmn-tck-runner-drools/README.md
```

Comment:
nitpick: I would simplify the property to "instructions.url". not all vendors might provide a runner, and instead just use some kind of tooling.